configs/advanced/component.yaml

```diff
@@ -47,6 +47,8 @@ proxy:
   maxDimension: 32768 # Maximum dimension of vector
   maxShardNum: 256 # Maximum number of shards in a collection
   maxTaskNum: 1024 # max task number of proxy task queue
+  bufFlagExpireTime: 3600 # second, the time to expire bufFlag from cache in collectResultLoop
+  bufFlagCleanupInterval: 600 # second, the interval to clean bufFlag cache in collectResultLoop
```

Comment:
@xiaofan-luan  Is it proper to put params here?

Reply:
sure we can~

Reply:
sure, we can leave it here